prometheus-formula/prometheus-formula.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu May  5 08:03:42 UTC 2022 - Witek Bedyk <witold.bedyk@suse.com>
+
+- Version 0.6.2
+  * Allow prometheus-formula only for SUSE systems (bsc#1199149)
```

Comment:
This is not totally exact.
![Uploading image.png…]()

Reply:
That's a documentation bug. I'll provide a fix for that.
We support only SUSE for the monitoring server.

Reply:
Thanks to clarify, we will update the test suite accordly.